package/CHANGELOG

```diff
@@ -17,7 +17,7 @@ mm/dd/yy richardjgowers, kain88-de, lilyminium, p-j-smith, bdice, joaomcteixeira
          PicoCentauri, davidercruz, jbarnoud, RMeli, IAlibay, mtiberti, CCook96,
          Yuan-Yu, xiki-tempula, HTian1997, Iv-Hristov, hmacdope, AnshulAngaria,
          ss62171, Luthaf, yuxuanzhuang, abhishandy, mlnance, shfrz, orbeckst,
-         wvandertoorn
+         wvandertoorn, cbouy
```

Comment:
Could you please update the CHANGELOG to also say that #2504 is fixed?

Reply:
This wasn’t in the scope of the original PR, but if you do, could you add a test that the verbose keyword does actually toggle the ProgressBar?

Reply:
Done !